### ISSUE TITLE
Request: Update x11vnc 0.9.13 -> 0.9.15

### ISSUE THREAD

**Issue by OPNA2608** [NixOS/nixpkgs#48263]
The last version of `x11vnc` released by the [original author](http://www.karlrunge.com/x11vnc), 0.9.13, is from September 2011 or so.

Since then, it has been developed further [on GitHub](https://github.com/LibVNC/x11vnc) by the community, amongst other things fixing an old, frequent and annoying
`*** stack smashing detected ***: <unknown> terminated`
exception that quits the running session and appears on several distributions still using the original release 0.9.13 (including NixOS) until it has been manually fixed by the package maintainers (in some cases).
[in openSUSE](https://bugzilla.suse.com/show_bug.cgi?id=987182)
[in Debian](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=859213)
[in Ubuntu 18.04](https://bugs.launchpad.net/ubuntu/+source/x11vnc/+bug/1686084)

I have been trying my luck with modifying the existing Nix expression to pull `x11vnc` and `libvncserver` from the Github repos instead, but I'm nowhere near experienced enough with Nix to provide anything working yet.

tl;dr Would it be possible to upgrade `x11vnc` to [this newer release](https://github.com/LibVNC/x11vnc/releases/tag/0.9.15)?
